allennlp/tests/fixtures/bert/bert_pooler.jsonnet

```diff
@@ -0,0 +1,53 @@
+{
+    "dataset_reader": {
+        "lazy": false,
+        "type": "bert_classification_test",
+        "tokenizer": {
+            "word_splitter": "bert-basic"
+        },
+        "token_indexers": {
+            "bert": {
+                "type": "bert-pretrained",
+                "pretrained_model": "allennlp/tests/fixtures/bert/vocab.txt"
+            }
+        }
+    },
+    "train_data_path": "doesn't matter, we're using precanned examples",
+    "validation_data_path": "also doesn't matter",
+    "model": {
+        "type": "basic_classifier",
+        "text_field_embedder": {
+            "allow_unmatched_keys": true,
+            "embedder_to_indexer_map": {
+                "bert": ["bert", "bert-offsets"],
+            },
+            "token_embedders": {
+                "bert": {
+                    "type": "bert-pretrained",
```

Comment:
It's worthwhile to include the options to configure training in the same manner as `bert_for_classification.jsonnet`.  To fine tune from the top layer it's necessary to also add

```
"requires_grad": false,
"top_layer_only": true,
```

Reply:
:+1: